From d0d3f35430b0cf4e1daa0e63179c86e4264c99e6 Mon Sep 17 00:00:00 2001
From: Paul Naoumenko <paulx.naoumenko@intel.com>
Date: Thu, 17 Oct 2013 18:21:41 +0200
Subject: [PORT FROM MAIN]wifi: Removing useless define

BZ: 125764

Change-Id: Ie41cd50ca0ce19eae01ea2a4b309fb54a8ff19b4
Orig-Change-Id: Ia370c22e0effeaece75558008c3a4334babd613d
Category: device enablement
Domain: CWS.WIFI-Common
Origin: internal
Upstream-Candidate: yes
Signed-off-by: Quentin Casasnovas <quentinx.casasnovas@intel.com>
---
 wifi/wifi.c    |    3 ---
 wifi/wifi_bc.c |    1 -
 2 files changed, 0 insertions(+), 4 deletions(-)

diff --git a/wifi/wifi.c b/wifi/wifi.c
index 952caf2..2675279 100644
--- a/wifi/wifi.c
+++ b/wifi/wifi.c
@@ -16,8 +16,6 @@
 
 #include "wifi.h"
 
-#define WIFI_DRIVER_LOADER_DELAY	1000000
-
 /*
  * This gets defined by the script load_bcmdriver in
  * vendor/intel/common/wifi/bcm_specific/
@@ -75,7 +73,6 @@ int wifi_load_driver()
         return -1;
 
     if (strcmp(FIRMWARE_LOADER,"") == 0) {
-        /* usleep(WIFI_DRIVER_LOADER_DELAY); */
         property_set(DRIVER_PROP_NAME, "ok");
     }
     else {
diff --git a/wifi/wifi_bc.c b/wifi/wifi_bc.c
index ff93324..3db92c7 100644
--- a/wifi/wifi_bc.c
+++ b/wifi/wifi_bc.c
@@ -81,7 +81,6 @@ static char primary_iface[PROPERTY_VALUE_MAX];
 // sockets is in
 
 #define WIFI_TEST_INTERFACE             "sta"
-#define WIFI_DRIVER_LOADER_DELAY        1000000
 
 static const char IFACE_DIR[]           = "/data/system/wpa_supplicant";
 
-- 
1.7.4.1

